c_reference/include/utils.h

```diff
@@ -47,15 +47,15 @@ void v_sigmoid(const float* const vec, const unsigned len, float* const ret) {
 }
 
 void v_tanh(const float* const vec, const unsigned len, float* const ret) {
-  for (unsigned i = 0; i < len; i++) ret[i] = tanhyperbolic(vec[i]);
+  for (unsigned i = 0; i < len; i++) ret[i] = tanh(vec[i]);
```

Comment:
we are using tanhyperbolic defined in utils.h file, instead of tanh from math.h, please revert this change